app/views/users/_audit_summary_query_builder.html.erb

```diff
@@ -0,0 +1,3 @@
+<div id="audit_summary_query_builder" class="input-group">
+  <%= text_f(f, :mail_query, {:label => '', :help_inline => _('Build a query for %{mailer}') % {:mailer => mailer.humanize }, :id => 'search',:class => "autocomplete-input", :data => {:url => auto_complete_search_audits_path} }) %>
```

Comment:
until mailer names are properly i18ned, use humanize.downcase so it isn't capitalised halfway through the sentence